src/staticanalysis/bot/static_analysis_bot/config.py

```diff
@@ -82,6 +94,19 @@ def setup(self, app_channel, cache_root, publication, allowed_paths):
         assert all(map(lambda p: isinstance(p, str), allowed_paths))
         self.allowed_paths = allowed_paths
 
+        # Set different info for Coverity
+        if cov_config is not None:
+            self.cov_analysis_url = cov_config[
```

Comment:
Please use `cov_config.get('XXX')` here.

I know it's automatically formatted, but it's really weird looking ! We support up to 120 characters on a line